src/main/java/com/google/step/finscholar/servlets/CollegeServlet.java

```diff
@@ -34,20 +34,15 @@
 @WebServlet("/college-data")
 public class CollegeServlet extends HttpServlet {
   private Firestore database;
-  private Gson gson;
+  private Gson gson = new Gson();
 
   @Override
-  public void init(ServletConfig config) throws ServletException {
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     try {
       database = FirestoreClient.getFirestore(FirebaseAppManager.getApp());
```

Comment:
Maybe get rid of class instance variable?

Reply:
Yeah good idea.